test/server/cards/06-CotE/GloriousVictory.spec.js

```diff
@@ -0,0 +1,131 @@
+describe('Glorious Victory', function() {
+    integration(function() {
+        describe('Glorious Victory', function() {
+            beforeEach(function() {
+                this.setupTest({
+                    phase: 'conflict',
+                    player1: {
+                        inPlay: ['shinjo-altansarnai', 'moto-juro'],
+                        hand: ['glorious-victory', 'captive-audience']
+                    },
+                    player2: {
+                        inPlay: ['adept-of-the-waves'],
+                        provinces: ['shameful-display', 'public-forum', 'endless-plains']
+                    }
+                });
+                this.shinjoAltansarnai = this.player1.findCardByName('shinjo-altansarnai');
+                this.motoJuro = this.player1.findCardByName('moto-juro');
+                this.gloriousVictory = this.player1.findCardByName('glorious-victory');
+                this.adeptOfTheWaves = this.player2.findCardByName('adept-of-the-waves', 'play area');
+                this.shamefulDisplay = this.player2.findCardByName('shameful-display', 'province 1');
+                this.publicForum = this.player2.findCardByName('public-forum');
+                this.endlessPlains = this.player2.findCardByName('endless-plains');
+                this.noMoreActions();
+            });
+
+            it('should trigger on breaking a province during a military conflict', function() {
+                this.initiateConflict({
+                    type: 'military',
+                    province: this.shamefulDisplay,
+                    attackers: [this.shinjoAltansarnai],
+                    defenders: [],
+                    jumpTo: 'afterConflict'
+                });
+                this.player1.clickPrompt('No');
+                expect(this.player1).toHavePrompt('Triggered Abilities');
+                expect(this.player1).toBeAbleToSelect(this.gloriousVictory);
+                expect(this.player1).toBeAbleToSelect(this.shinjoAltansarnai);
+                this.player1.clickCard(this.gloriousVictory);
+                expect(this.shinjoAltansarnai.isHonored).toBe(true);
+                expect(this.motoJuro.isHonored).toBe(true);
+                expect(this.adeptOfTheWaves.isHonored).toBe(false);
+            });
+
+            it('should not trigger on breaking a province during a political conflict', function() {
+                this.initiateConflict({
+                    type: 'political',
+                    province: this.shamefulDisplay,
+                    attackers: [this.shinjoAltansarnai],
+                    defenders: [],
+                    jumpTo: 'afterConflict'
+                });
+                this.player1.clickPrompt('No');
+                expect(this.player1).not.toHavePrompt('Triggered Abilities');
```

Comment:
Again, not a big deal, but I would add this line at the end of this test:
```
expect(this.player1).toHavePrompt('Air Ring');
```
This is just to check that you have passed the 'break province' step and Glorious Victory has not triggered.

Reply:
Added